libs/designsystem/header/src/header.component.html

```diff
@@ -7,16 +7,17 @@
 </div>
 
 <div class="container">
+    <!-- Title and value - title: -->
+  <h1 *ngIf="!!title && !!value" #titleElement class="title kirby-text-medium">
+    {{ title }}
+  </h1>
   <div class="title-container">
     <!-- Only title, no value: -->
     <h1 *ngIf="!!title && !value" #titleElement class="title" [kirbyFitHeading]="fitHeadingConfig">
       {{ title }}
     </h1>
   
-    <!-- Title and value: -->
-    <h1 *ngIf="!!title && !!value" #titleElement class="title kirby-text-medium">
-      {{ title }}
-    </h1>
+    <!-- Title and value - value: -->
```

Comment:
Just a quick question - is this comment still descriptive of the code below? :)

Reply:
@mark-drastrup Yes it is. Might not be perfect, but it refers to the scenario where we have BOTH a title AND a value.
And those 2 elements are now split at different locations in the markup (the bulk of this PR), so it was an attempt at stating this:

Top of container element:
  `<!-- Title and value - title: -->`
  
  and further down in the markup:
 ` <!-- Title and value - value: -->`